[OSDOCS-12177] removing firewall prereqs

aldiazRH opened the pull request:
[OSDOCS-12177] removing firewall prereqs

<!--- If your changes apply to the latest release and/or in-development version of OpenShift, open your PR against the `main` branch.

* For more details about the information requested in this template, see:
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/create_or_edit_content.adoc#submit-PR --->

Version(s):
4.17+

Issue:
https://issues.redhat.com/browse/OSDOCS-12177

Link to docs preview:
https://82742--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_planning/aws-ccs.html
https://82742--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.html
https://82742--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-aws-prereqs.html

QE review:
- [x] QE has approved this change.
<!--- QE approval is required to merge a PR except for changes that do not impact the meaning of the docs. --->

Additional information:
<!--- Optional: Include additional context or expand the description here.--->

<!--- After you open your PR, ask for review from the OpenShift docs team:
  For community authors: Tag @openshift/team-documentation in a GitHub comment.--->

michaelryanmcneill reviewed (changes requested):
It appears that `quayio-production-s3.s3.amazonaws.com` still exists in the HCP list of prereqs and also needs to be removed from there.

aldiazRH commented:
> It appears that `quayio-production-s3.s3.amazonaws.com` still exists in the HCP list of prereqs and also needs to be removed from there.

Updated and fixed, thanks @michaelryanmcneill

michaelryanmcneill reviewed (approved):
LGTM

aldiazRH commented:
/label peer-review-needed

GroceryBoyJr commented:
/label peer-review-in-progress
/remove-label peer-review-needed
/assign GroceryBoyJr

GroceryBoyJr commented:
/label peer-review-done
/remove-label peer-review-in-progress
/unassign GroceryBoyJr

GroceryBoyJr reviewed (approved):
LGTM

aldiazRH commented:
/label merge-review-needed

mburke5678 commented:
/cherrypick enterprise-4.17

mburke5678 commented:
/cherrypick enterprise-4.18
